components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+const SearchBarContainer = styled.form`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 0.5rem;
+`;
+
+const SearchInput = styled.input`
+  padding: 0.3rem;
+  font-size: 0.9rem;
+`;
+
+const SearchButton = styled.button`
+  font-size: 1rem;
+`;
+
+export default function SearchBar({ setSearchTerm }) {
+  return (
+    <>
+      <SearchBarContainer id="form">
+        <SearchInput
+          onChange={(event) => setSearchTerm(event.target.value)}
```

Comment:
We could potentially lowercase the search term here. That would save us needing to do it 3 times in the foundBooks filter in index.js

Reply:
Thanks, adapted